src/sync.rs

```diff
@@ -659,10 +643,7 @@ mod tests {
         };
 
         let fiona = &tcm.fiona().await;
-        tcm.exec_securejoin_qr(fiona, alice, &qr).await;
-        let msg_id = alice.send_sync_msg().await?;
-        // The QR code token was already synced before.
-        assert!(msg_id.is_none());
+        tcm.exec_securejoin_qr(fiona, alice2, &qr).await;
```

Comment:
Amazing that this works. Alice2 doesn't have the group in the db, only the QR code token. Need to add more checks that the group is created with expected params

Reply:
It didn't really work, `exec_securejoin_qr()` doesn't guarantee the protocol succeeds:
`alice WARN: src/securejoin.rs:418: Ignoring vg-request-with-auth message: unknown grpid BjI8zTDM9E7.`
Now i've added a minimal check that Fiona joins successfully.